Jenkinsfile

```diff
@@ -132,19 +132,61 @@ pipeline {
                         sh 'poetry run sphinx-build -W -b html docs docs/_build/html'
                     }
 
+                    dir('external-docs'){
+                        sh "python3 -m mkdocs build"
+                    }
+
                     stash name: 'docs', includes: '**/docs/_build/html/**/*'
```

Comment:
You could have included the mkdocs stuff in the `docs` stash if you'd wanted, but it isn't the end of the world.

I think something like `includes: '**/docs/_build/html/**/*,**/external-docs/site/**/*'` would have done it.

Reply:
Also, since you've added a new stash, are you making sure it's archived as an artifact in the final part?